src/dataflow/src/source/s3.rs

```diff
@@ -74,6 +74,8 @@ type Out = MessagePayload;
 struct InternalMessage {
     record: Out,
 }
+/// Size of data chunks we send to dataflow
+const CHUNK_SIZE: usize = 4096;
```

Comment:
Elsewhere we're using buffers of 8KiB, you could do the same here, but not required.

Reply:
Where is that? I can't find it anywhere.

Reply:
For example: https://github.com/TimelyDataflow/timely-dataflow/blob/a4106707af386a471dd7eef33d60f3fca4720271/timely/src/dataflow/channels/mod.rs#L31